docs/v3-api.yaml

```
@@ -809,7 +809,13 @@ paths:
             default: true
           in: query
           name: include-unicode
-          description: Unicode絵文字を含ませるかどうか
+          description: (Duplicated) Unicode絵文字を含ませるかどうか
```

```suggestion
          deprecated: true
          description: |-
            Unicode絵文字を含ませるかどうか
            Deprecated: typeクエリを指定しない場合はすべて取得できるため、それを利用してください
```
https://swagger.io/specification/#operation-object
`deprecated`が存在するので使ったほうが良いですね
あとは、ほかに合わせるとよさそうです
https://github.com/traPtitech/traQ/blob/e5d819baea72a65cb8b3ea59bb18d7dd034ccd0e/docs/v3-api.yaml#L4651-L4657